test/RequestGroupTest.cc

```diff
@@ -39,6 +39,35 @@ void RequestGroupTest::testGetFirstFilePath()
 
   CPPUNIT_ASSERT_EQUAL(std::string("/tmp/myfile"), group.getFirstFilePath());
 
+  // test file renaming
```

Comment:
In my opinion, these tests should go their own function, say, testTryAutoFileRenaming().